.github/workflows/ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
     - name: Set up Go 1.x
       uses: actions/setup-go@v2
       with:
-        go-version: 1.15
+        go-version: ^1.15
```

Comment:
Sorry one more thing I missed, since we no require 1.16 go version we should setup our CI to be go 1.16 onwards for all jobs in this file.

Reply:
I think this is the most rigorous, I changed it to 1.16+.